packages/workflow/src/Expression.ts

```diff
@@ -40,13 +41,9 @@ export const isExpressionError = (error: unknown): error is ExpressionError =>
 export const isTypeError = (error: unknown): error is TypeError =>
 	error instanceof TypeError || (error instanceof Error && error.name === 'TypeError');
 
-// Set it to use double curly brackets instead of single ones
-tmpl.brackets.set('{{ }}');
-
-// Make sure that error get forwarded
-tmpl.tmpl.errorHandler = (error: Error) => {
+setErrorHandler((error: Error) => {
```

Comment:
What's the difference between this and L341? Do we still need this?

Reply:
I kept the same behaviour in `tournament` as `tmpl` and required some kind of error handler. In both cases errors will just get eaten if nothing is provided. I'll add that comment back to make it clearer.